mlflow/experiments.py

```diff
@@ -108,3 +109,15 @@ def rename_experiment(experiment_id, new_name):
     store = _get_store()
     store.rename_experiment(experiment_id, new_name)
     print("Experiment with id %s has been renamed to '%s'." % (experiment_id, new_name))
+
+
+@commands.command("csv")
+@EXPERIMENT_ID
+@click.option("--filename_csv", type=click.STRING, required=True)
```

Comment:
Just change this flag name to `--filename` since CSV is redundant under the command

Reply:
Done